src/main/java/com/saralein/server/connection/ConnectionSocket.java

```diff
@@ -11,6 +15,21 @@ public ConnectionSocket(Socket socket) {
         this.socket = socket;
     }
 
+    public String read() throws IOException {
+        StringBuilder request = new StringBuilder();
+
+        InputStreamReader in = new InputStreamReader(socket.getInputStream());
+        BufferedReader bufferedReader = new BufferedReader(in);
+
+        String str;
+
+        while ((str = bufferedReader.readLine()) != null && !str.equals("")) {
```

Comment:
is there a function on str, that you can use to see if the string is empty instead of an equals on "", not sure if there is, in C# there's `String.IsNullOrEmpty(str)`, just wondering if there's anything similar